compiler/src/dmd/lexer.d

```diff
@@ -105,13 +111,12 @@ class Lexer
      *  doDocComment = handle documentation comments
      *  commentToken = comments become TOK.comment's
      *  errorSink = where error messages go, must not be null
-     *  vendor = name of the vendor
-     *  versionNumber = version of the caller
+     *  compileEnv = version, vendor, date, time, etc.
      */
     this(const(char)* filename, const(char)* base, size_t begoffset,
         size_t endoffset, bool doDocComment, bool commentToken,
         ErrorSink errorSink,
-        const(char)[] vendor = "DLF", uint versionNumber = 1) pure scope
+        const CompileEnv* compileEnv = null) pure scope
```

Comment:
I would say, just remove the default parameter.

Reply:
ok